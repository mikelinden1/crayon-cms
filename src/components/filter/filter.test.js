import Filter from './filter';

const props = {
    filterVals: { name: 'Mike' },
    filters: [{ name: 'name', label: 'Name' }],
    actions: {
        fetchDatasource: () => {},
        setFilter: () => {}
    }
};

it('Filter matches snapshot', () => {
    const wrapper = shallow(<Filter {...props} />);
    expect(wrapper).toMatchSnapshot();
});