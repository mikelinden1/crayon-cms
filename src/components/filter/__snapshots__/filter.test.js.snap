// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter matches snapshot 1`] = `
<DropdownField
  actions={
    Object {
      "fetchDatasource": [Function],
      "setFilter": [Function],
    }
  }
  key="name-filter"
  label="Name"
  name="name"
  onChange={[Function]}
  placeholder="Name Filter"
  value="Mike"
/>
`;
